README.md

```diff
@@ -1,24 +1,23 @@
 Use the [Webnative SDK](https://github.com/fission-codes/webnative#readme) with a blockchain wallet. Access your personal encrypted file system with your wallet keys.
 
-
 ## Usage
 
 Uses Ethereum by default with `window.ethereum` as the provider. Currently only works with MetaMask because it's the only wallet with encryption and decryption.
 
 ```ts
-import * as walletauth from "webnative-walletauth"
-import { AppScenario } from "webnative"
+import * as walletauth from "webnative-walletauth";
+import { Scenario } from "webnative";
```

Comment:
@avivash `AppScenario` was correct as before, `Scenario` is a slightly different object used for permissioned apps (see https://webnative.fission.app/modules.html#permissionedApp)

Reply:
> @avivash `AppScenario` was correct as before, `Scenario` is a slightly different object used for permissioned apps (see https://webnative.fission.app/modules.html#permissionedApp)

strange 🤔, it didn't seem to work when I tried with `AppScenario`. Is that possible in an unreleased version? 
<img width="910" alt="image" src="https://user-images.githubusercontent.com/1179291/188910163-4c3fa86b-b339-4080-9a72-d6bcbb60ecb3.png">

Reply:
Hmm weird, it's in 0.34.x https://webnative.fission.app/enums/AppScenario.html
Does it keep saying that after reloading your TS language server?

Reply:
> Hmm weird, it's in 0.34.x https://webnative.fission.app/enums/AppScenario.html Does it keep saying that after reloading your TS language server?

Yep, still doesn't work after a TS server restart, unfortunately

Reply:
I'll push the readme update separately 👌🏼

Reply:
Oh nevermind, I see you already did! 🙌🏼

Reply:
I already changed it back, all good 👍